### PR TITLE
chore(document-list-styles): align css

### PR DESCRIPTION
**Pull request by mabaasit**
Remove unwanted margins from the document list in the pipeline preview. Also aligns the document views on crud (ListView, JsonView and TableView)

<details>
<summary>Preview</summary>

Before: 
image
After:
image

CRUD View:
image

</details>

### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass-crud/src/components/document-list.jsx [hunk 1]
```diff
@@ -23,6 +25,20 @@ import {
 import './index.less';
 import './ag-grid-dist.css';
 
+
+const listAndJsonStyles = css({
```
Comment (mcasimir) [inline packages/compass-crud/src/components/document-list.jsx:29]: can you take screenshots of these changes? what are those for? thanks!

Comment (mabaasit) [reply packages/compass-crud/src/components/document-list.jsx:29]: added in preview. can you check it?

Comment (mcasimir) [reply packages/compass-crud/src/components/document-list.jsx:29]: Yeah, is a bit too narrow to get an idea of all the changes, I see a lot of spaces being adjusted here and there in crud in the crud view, is all we changed already in those screenshots?

Btw if is too much work no worries I can also pull this branch and check it later!

Comment (mcasimir) [reply packages/compass-crud/src/components/document-list.jsx:29]: Ohhh sorry you added it, thanks!!

**Review by mcasimir — APPROVED** [review]
👨‍🎨👨‍🎨👨‍🎨

**Pull request merged**